docs/Jupyter_and_colab.rst

```diff
@@ -12,21 +12,87 @@ To install the TensorboardPlugin3D python package inside the notebook:
 
 Usage
 -----
-The plugin currently looks at the first image available in the log directory
-and visualizes it. If two images are available the second image is assumed to
-be a label. Each image or image/label pair should be written to its own log
-directory. The directory to be used can then be set and visualized with the
+The plugin currently looks at the most recent image available in the log directory
+and visualizes it. If two images are available for the most recent run and one
+of them is tagged "label" it will be treated as such. Each image or image/label
+pair can be written to its own log directory to make it simple to switch
+between views. The directory to be used can then be set and visualized with the
 tensorboard widget within either JupyterLab or Colab.
 
+To break down some examples of options for writing out images we can look at
+the UNet notebook:
+
+.. image:: images/unet_notebook.png
+   :alt: UNet Notebook Cells
+
+In the first cell we are selecting the data that we would like to view, in this
+case the first input image and label as well as the first output model. The
+output model is then written out in the final line of the first cell shown:
+
     .. code-block::
 
-        tb_dir = os.path.join(root_dir, logdir_to_use)
-        %load_ext tensorboard
-        %tensorboard --logdir=$tb_dir
+        plot_2d_or_3d_image(data=val_outputs, step=0, writer=SummaryWriter(log_dir=log_dir), frame_dim=-1, tag="image")
+
+The call to ``plot_2d_or_3d_image`` takes several parameters:
+
+    - *data*
+        Target data to be plotted as image on the TensorBoard. The data is
+        expected to have "NCHW[D]" dimensions and only plot the first in the
+        batch. This shape is important and required to write out the full 3D
+        image.
+    - *step*
+        Current step to plot in a chart.
+    - *writer*
+        Specify a SummaryWriter to plot the image. You should create a new
+        SummaryWriter object for each new image or image/label pair that you
+        create so that they are grouped together.
+    - *frame_dim*
```

Comment:
Calling this "frame" is confusing to me.  Later in the doc you specify -1 for frame.  Is this meant to be the "slice" dimension or the depth dim or ????

Reply:
Can we call this "Depth_dim" instead of frame_dim?    I think that emphasizes and corresponds to the 3D nature of this plugin more.   MONAI has some inconsistencies/poor-terms since it was mainly developed by computer vision people instead of medical people at the start.   :(

Reply:
Ah - I see - never mind.   Yup, frame_dim it is.  :(